examples/README.md

```diff
@@ -480,6 +482,72 @@ python3 -m http.server --directory examples/wasm
 ruby -run -ehttpd examples/wasm
 ```
 
+### Optimizing
+
+On the web, it's useful to reduce the size of the files that are distributed.
+With rust, there are many ways to improve your executable sizes.
+Here are some.
+
+#### 1. Tweak your `Cargo.toml`
+
+Add this to your `Cargo.toml`:
```

Comment:
this part could use a link to https://doc.rust-lang.org/cargo/reference/profiles.html